test/unit/testWillCodicils.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+const initSteps = require('app/core/initSteps');
+const chai = require('chai');
+const expect = chai.expect;
+const assert = require('chai').assert;
+const steps = initSteps([`${__dirname}/../../app/steps/action/`, `${__dirname}/../../app/steps/ui`]);
+const WillCodicils = steps.WillCodicils;
+const json = require('app/resources/en/translation/will/codicils');
+
+describe('WillCodicils', () => {
+    describe('getUrl()', () => {
+        it('should return the correct url', (done) => {
+            const url = WillCodicils.constructor.getUrl();
+            expect(url).to.equal('/will-codicils');
+            done();
+        });
+    });
+
+    describe('handlePost()', () => {
+        let ctx;
+        let errors;
+        let formdata;
+        let session;
+        let hostname;
+        let featureToggles;
+
+        it('should return the ctx with the deceased married status and the screening_question feature toggle', (done) => {
+            ctx = {
+                codicils: 'Yes'
+            };
+            errors = {};
+            [ctx, errors] = WillCodicils.handlePost(ctx, errors, formdata, session, hostname, featureToggles);
+            expect(ctx).to.deep.equal({
+                codicils: 'Yes',
+                isToggleEnabled: false
+            });
+            done();
+        });
+    });
+
+    describe('nextStepOptions()', () => {
```

Comment:
This method should also have a test for the case where the toggle is enabled as this will output a different return value

Reply:
Done